internal/k8s/pod.go

```diff
@@ -105,6 +105,20 @@ func NodeIDFromPod(pod *v1.Pod) NodeID {
 	return NodeID(pod.Spec.NodeName)
 }
 
+func RestartsFromPod(pod *v1.Pod) int {
+	var restarts int32 = 0
+	for _, cStatus := range pod.Status.ContainerStatuses {
+		if cStatus.Name == "tilt-synclet" { // this should be a constant but there's nowhere good to put it :(
+			continue
+		}
+		// If pod has multiple containers, take the max # of restarts
```

Comment:
what do you think about summing instead of maxing?

Reply:
ah yyyyep that's totally the behavior of `kubectl get pods` isn't it? will do.